src/dataset/movie_rating.py

```
@@ -0,0 +1,189 @@
+        os.path.dirname(
+            os.path.dirname(os.path.abspath(__file__))))
+)
+
+import pandas as pd
+import numpy as np
+from konlpy.tag import Okt
+import torch
+import torch.nn as nn
+import torch.nn.utils.rnn as rnn_utils
+from sklearn.feature_extraction.text import TfidfVectorizer
+from sklearn.preprocessing import LabelEncoder
```

Ruff (0.12.2)

`sklearn.preprocessing.LabelEncoder` imported but unused

Remove unused import: `sklearn.preprocessing.LabelEncoder`

(F401)